engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueBlocksNode.java

```diff
@@ -86,6 +87,10 @@ public OpaqueBlocksNode(Context context) {
         renderQueues = context.get(RenderQueuesHelper.class);
         worldProvider = context.get(WorldProvider.class);
 
+        // Added before instantiating WireframeTrigger so it can remove this state change if needed - see enableWireframe()
```

Comment:
Let's rewrite this as:

```
// IF wireframe is enabled the WireframeTrigger will remove the face culling state change 
// from the set of desired state changes.
// The alternative would have been to check here first if wireframe mode is enabled and *if not*
// add the face culling state change. However, if wireframe *is* enabled, the WireframeTrigger
// would attempt to remove the face culling state even though it isn't there, relying on the
// quiet behaviour of Set.remove(nonExistentItem). We therefore favored the first solution.
```

And please remember to add the same text to `OpaqueObjectNode`!